c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -819,10 +823,6 @@ class THCCachingAllocator {
 
 THCCachingAllocator caching_allocator;
 
-static void CudaCachingDeleter(void* ptr) {
```

Comment:
Would I be correct if I assume this is a duplication of the `raw_delete` method?

Reply:
Yes, it is a duplicate that is why I removed it.